app/screens/backup/TwelveWordsBackup.js

```diff
@@ -136,45 +128,48 @@ class TwelveWordsBackup extends Component<Props, State> {
-          paper and store the paper in a safe place. Alternatively, print it, or copy & paste it into your password manager.
-        </HeaderExplanation>
-        <TwelveWordsContainer>
-          {this.twelveWords.map((word: string, index: number) => (
-            <WordWrapper key={word}>
-              <IndexWrapper>
-                <Index>{`${index + 1}`}</Index>
-              </IndexWrapper>
-              <Text>{word}</Text>
-            </WordWrapper>
-          ))}
-        </TwelveWordsContainer>
-        <NotificationSection>
-          {isTwelveWordsCopied && (
-            <React.Fragment>
-              <Notification>The list has been copied to your clipboard. Paste it into your password manager.</Notification>
-              <Notification>Next step, we will test you with a small little game.</Notification>
-            </React.Fragment>
-          )}
-        </NotificationSection>
-        <ButtonsRow>
-          <LeftButtonsContainer>
-            <SmButton text="Print" theme="green" onPress={this.print12Words} style={{ width: 150, marginRight: 20 }} />
-            <SmButton text="Copy" theme="green" onPress={this.copy12Words} style={{ width: 150 }} />
-          </LeftButtonsContainer>
-          <SmButton text="Test Me" theme="orange" onPress={this.navigateToTestMe} style={{ width: 150 }} />
-        </ButtonsRow>
-        <ActionLink onClick={this.learnMoreAboutPaperBackup}>Learn more about paper backup</ActionLink>
-      </Wrapper>
+      <WrapperWith2SideBars width={920} height={480} header="YOUR 12 WORDS BACKUP">
+        <HorizontalBarWrapper>
+          <HorizontalBar src={smallHorizontalSideBar} />
+        </HorizontalBarWrapper>
+        <TextWrapper>
+          <Text>
+            A paper backup is a numbered list of words written down on a paper Write down or print this numbered word list and store the paper in a a safe place, or copy & paste it
+            into your password manager
+          </Text>
+        </TextWrapper>
+        <MiddleSectionRow>
+          <ButtonsSection>
+            <Button onClick={this.print12Words} text="PRINT WORDS" width={172} isPrimary={false} style={{ marginBottom: 12 }} />
+            <Button onClick={this.copy12Words} text="COPY WORDS" width={172} isPrimary={false} style={{ marginBottom: 12 }} />
+            {isTwelveWordsCopied && <GreenText>Copied to clipboard</GreenText>}
+          </ButtonsSection>
+          <WordsSection>
+            {this.twelveWords.map((word: string, index: number) => (
+              <WordWrapper key={word}>
+                <IndexWrapper>
+                  <Index>{`${index + 1}`}</Index>
+                </IndexWrapper>
+                <WordContainer>
+                  <Text>{word}</Text>
+                </WordContainer>
+              </WordWrapper>
+            ))}
+          </WordsSection>
+        </MiddleSectionRow>
+        <BottomSection>
```

Comment:
as I pointed earlier this composition is redundant and looks very amateur, especially with this plaster style={{ paddingTop: 26 }}